dali/base/dadfs.cpp

```diff
@@ -9162,10 +9162,10 @@ class CInitGroups
         if (!newClusterGroup) // both null
             return true;
         // see if identical
-        const char *oldKind = oldClusterGroup?oldClusterGroup->queryProp("@kind"):NULL;
```

Comment:
It is, it's basically saying if both null - then they match - if either null then mismatch.